
human_trajectory/Trajectory trajectory
---
float64 spatial_dist
float64 temp_nov
float64 overall_nov
